exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -45,6 +45,8 @@ drill.client: {
   supports-complex-types: true
 }
 
+drill.home: "/tmp"
```

Comment:
Drill home is /tmp? Please explain...

From below, it seems that "drill.home" means the Drill home directory in DFS. (The problem is, we already use $DRILL_HOME to mean the location of the Drill install.)

Maybe change this to "drill.dfs.home" or "drill.dfs-home" to make it clear that this is for DFS.